### PR TITLE
🚀 Release v1.7.1

### PR DESCRIPTION
**Pull request by chandankumar4**
/area release

### CODE REVIEW

File: CHANGELOG/v1.7.1.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.26.x -> v1.29.x
```
Comment (adilGhaffarDev) [inline CHANGELOG/v1.7.1.md:3]:
```suggestion
- Management Cluster: v1.26.x -> v1.30.x
```

File: CHANGELOG/v1.7.1.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- Workload Cluster: v1.24.x -> v1.29.x
```
Comment (adilGhaffarDev) [inline CHANGELOG/v1.7.1.md:4]:
```suggestion
- Workload Cluster: v1.24.x -> v1.30.x
```

File: CHANGELOG/v1.7.1.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Highlights
+
+* CAPD: Verify lb config after writing it (#10461)
```
Comment (adilGhaffarDev) [inline CHANGELOG/v1.7.1.md:10]:
```suggestion
* Kubernetes v1.30 is now supported
```
I don't think we need to put #10461 in the highlights.

**Review by adilGhaffarDev — COMMENTED** [review]
We need to do some manual changes to inform about the support of Kubernetes v1.30.

**Comment by adilGhaffarDev** [comment]
cc: @chrischdi @sbueringer @fabriziopandini please take a look when you get time.

**Review by chrischdi — COMMENTED** [review]
Sorry for the cherry-picks but this one is missing currently:

- #10481

**Comment by sbueringer** [comment]
Overall looks good so far. Please just regenerate before you release and add the PRs that show up in the delta

**Review by Sunnatillo — COMMENTED** [review]
LGTM

File: CHANGELOG/v1.7.1.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+* Kubernetes v1.30 is now supported
+
+## Changes since v1.7.0
+## :chart_with_upwards_trend: Overview
+- 11 new commits merged
+- 2 bugs fixed 🐛
+
+## :bug: Bug Fixes
+- CAPD: Verify lb config after writing it (#10461)
+- e2e: Kubetest: also gather junit reports in case of errors observed from ginkgo (#10494)
```
Comment (willie-yao) [inline CHANGELOG/v1.7.1.md:19]:
```suggestion
- e2e: also gather junit reports in case of errors observed from ginkgo (#10494)
```

**Pull request merged**